build.yaml

```diff
@@ -110,3 +110,18 @@ menu:
       - entry: "Windows 10"
         build:
           path: "builds/windows/desktop/10/"
+  - entry: "Child builds"
+    build:
+      var_files:
+        - "vsphere.pkrvars.hcl"
+        - "build.pkrvars.hcl"
+        - "ansible.pkrvars.hcl"
+        - "proxy.pkrvars.hcl"
+        - "common.pkrvars.hcl"
+    submenu:
+      - entry: "Ubuntu Server 22.04 LTS child from VM Template"
+        build:
+          path: "builds/childs/template"
```

Comment:
Should we consider `base` and `clone` terminology? Where base is the parent and clone is the child? If so, adjust the directory structure?

Reply:
I prefer `base` and `clone` terminology.

Reply:
Done in commit d0f0f0646724ddca92b54c9566a9c33068f549dc